src/main/java/de/bsi/secvisogram/csaf_cms_backend/service/AdvisoryService.java

```diff
@@ -315,7 +315,7 @@ public String updateAdvisory(String advisoryId, String revision, CreateAdvisoryR
             if (canChangeAdvisory(oldAdvisoryNode, credentials)) {
 
                 if (changedCsafJson.getSummary() == null || changedCsafJson.getSummary().isBlank()) {
-                    throw new CsafException("Summary must not be empty", SummaryInHistoryEmpty, UNAUTHORIZED);
+                    throw new CsafException("Summary must not be empty", SummaryInHistoryEmpty, BAD_REQUEST);
```

Comment:
Could you improve the Test AdvisoryServiceTest.updateAdvisoryTest_noSummary to check the error status?

like in 

https://stackoverflow.com/questions/58269655/how-to-use-junit-5-assertions-to-check-whether-exception-message-starts-with-a

Reply:
Updated and also added that to the other tests of `updateAdvisory`.

I noticed that we actually get the `IdNotFoundException` thrown by the `CouchDbService` in `updateAdvisoryTest_badData` and `updateAdvisoryTest_invalidId` instead of the one thrown in `de/bsi/secvisogram/csaf_cms_backend/service/AdvisoryService.java:307`. I did not invest the time now to correct that for getting more expressive error messages. Should we open an issue for this?